fairscale/experimental/wgit/pygit.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+from pathlib import Path
+from typing import List
+
+import pygit2
+
+
+class PyGit:
+    def __init__(
+        self,
+        parent_path: Path,
+        gitignore: List = list(),
+        name: str = "user",
+        email: str = "user@email.com",
+    ) -> None:
+        """
+        PyGit class to wrap the wgit/.git repo and interact with the git repo.
+
+        Args:
+        parent_path: Has to be the full path of the parent!
```

Comment:
pls describe the meaning of this arg in addition of the important note here.

also document other args like gitignore, name, email.

maybe no need to have default values for name and email? Let caller always pass them explicitly?